kernel/src/engine/default/parquet.rs

```diff
@@ -261,6 +280,7 @@ mod tests {
             size: meta.size,
         }];
 
+        // TODO: add a test that uses predicate skipping?
```

Comment:
A new test case in read.rs covers row group skipping for both sync and default readers